src/rmarkdown/preview.ts

```diff
@@ -0,0 +1,357 @@
+
+import * as cp from 'child_process';
+import * as vscode from 'vscode';
+import * as fs from 'fs-extra';
+
+import { config, doWithProgress, getRpath, readContent, setContext } from '../util';
+import { extensionContext, tmpDir } from '../extension';
+import path = require('path');
+import crypto = require('crypto');
+
+class RMarkdownChild extends vscode.Disposable {
```

Comment:
Does it make sense to rename `RMarkdownChild` to `RMarkdownPreview` and `RMarkdownChildStore` to `RMarkdownPreviewStore`.

It seems to look more natural to have `RMarkdownPreviewManager` manage a collection of `RMarkdownPreview` stored in `RMarkdownPreviewStore`.